src/main/java/com/fasterxml/aalto/sax/SAXParserFactoryImpl.java

```diff
@@ -68,6 +69,8 @@ public boolean getFeature(String name)
             switch (stdFeat) {
             case IS_STANDALONE: // read-only, but only during parsing
                 return true;
+            case EXTERNAL_GENERAL_ENTITIES:
+                return !((Boolean) mStaxFactory.getProperty(AaltoInputProperties.P_RETAIN_GENERAL_ENTITIES)).booleanValue();
```

Comment:
Is this null safe? (value `null` does not become non-null Boolean)
I'd probably simplify this (given that missing means "false") to something like

    return Boolean.TRUE.equals(mStaxFactory.getProperty(....))